creation/interfaces-and-networks.md

```diff
@@ -692,3 +681,164 @@ documentation](https://github.com/openshift/sriov-network-operator/blob/master/d
 > into VMI guest. Because of that, when configuring SR-IOV operator
 > policies, make sure you define a pool of VF resources that uses
 > `driver: vfio`.
+> **Note:** Further instructions will assume the aforementioned operator
```

Comment:
The "we assume operator" note is in this "How to expose SR-IOV VFs to KubeVirt" but it is missing in the following in the following "Start an SR-IOV VM".

Assuming I'm not the only idiot not reading articles from start to the end, but just parsing the tree, would it make sense to keep the operator note above this first section?